src/viam/ur/module/ur_arm_state.cpp

```diff
@@ -101,7 +62,11 @@ std::unique_ptr<URArm::state_> URArm::state_::create(std::string configured_mode
                                                      const struct ports_& ports) {
     auto host = find_config_attribute<std::string>(config, "host").value();
 
-    auto resource_root = find_resource_root();
+    const auto module_executable_path = boost::dll::program_location();
+    const auto module_executable_directory = module_executable_path.parent_path();
+    auto resource_root = std::filesystem::canonical(module_executable_directory / k_relpath_bindir_to_datadir / "universal-robots");
```

Comment:
[note to self] in the previous pr we made it so `k_relpath_bindir_to_datadir` gets defined  to get cmake to specify this